Source/FullScreenPokemon.d.ts

```diff
@@ -1096,6 +1096,41 @@ declare module FullScreenPokemon {
     }
 
     /**
+     * The types of Pokemon that can be caught with different rods.
+     */
+    export interface IFishing extends IItemSchema {
+        /**
+         * The Pokemon that can be caught using an Old Rod.
+         */
+        old?: IWildPokemonSchema[];
+
+        /**
+         * The Pokemon that can be caught using a Good Rod.
+         */
+        good?: IWildPokemonSchema[];
+
+        /**
+         * The Pokemon that can be caught using a Super Rod.
+         */
+        super?: IWildPokemonSchema[];
+    }
+
+    /**
+     * The type of rod that is being used.
+     */
+    export interface IRod extends IItemSchema {
+        /**
+         * The type of rod used. Can be old, good, or super.
+         */
+        type: string;
```

Comment:
Yeah that's what I originally tried but it doesn't work. It gives an error "Type expected" as well as, for both the left and right hand sides, "the left-hand side of an arithmetic operation must be of type 'any', 'number', or an enum type".